### DISCUSSION TITLE
Having trouble updating via the server installation method

### DISCUSSION THREAD

**Discussion by chansters** [mediacms-io/mediacms#487] [Q&A]
**Describe the issue**
I'm trying to update my MediaCMS installation for the first time, following the server installation update instructions (using git pull) but I'm running into an error message 

`error: cannot open .git/FETCH_HEAD: Permission denied`

**To Reproduce**
Steps to reproduce the issue:
1. Log into my Ubuntu VM (Ubuntu 20.04.4 LTS (GNU/Linux 5.4.0-122-generic x86_64))
2. cd ../mediacms.io/mediacms/
3. source  /home/mediacms.io/bin/activate4. 
4. (mediacms.io) chris@mediacms:/home/mediacms.io/mediacms$ git pull
5. error: cannot open .git/FETCH_HEAD: Permission denied

**Expected behavior**
Expect the update process to happen as described in the instructions

**Environment (please complete the following information):**
 - OS: Ubuntu 20.04.4 LTS
 - Installation method: Single server install
 - Browser, if applicable

**Additional context**
First time trying to update since installation in December 2021.

**Comment by mgogoulos** [comment]
Hi, this `Permission denied` message indicates that you have cloned the repository as root and now you are trying to `git pull` as user. Su to root and try again!
